plugins/entity-feedback-backend/src/service/router.ts

```diff
@@ -167,6 +170,18 @@ export async function createRouter(
     res.json(ratings.filter(r => accessibleEntityRefs.includes(r.userRef)));
   });
 
+  router.get('/ratings/:entityRef/aggregate', async (req, res) => {
+    const entityRatings = (
+      await dbHandler.getRatingAggregate(req.params.entityRef)
+    ).reduce((ratings: Ratings, { rating, count }) => {
+      ratings[rating] = count;
+
+      return ratings;
+    }, {});
```

Comment:
hm I don't think we actually need a separate db method for this. We can probably reuse and aggregate the result from the current `dbHandler.getRatings()` call:
```suggestion
    const entityRatings = (
      await dbHandler.getRatings(req.params.entityRef)
    ).reduce((ratings: Ratings, { rating }) => {
      ratings[rating] = ratings[rating] ?? 0;
      ratings[rating]++;

      return ratings;
    }, {});
```

Reply:
Thanks @kuangp ! I couldn't find a good way to reuse the other endpoint but this works indeed.
I have updated it in the PR